[2024-05-07 19:29:12,729] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: research_scraper.research_scraper manual__2024-05-07T19:29:10.005131+00:00 [queued]>
[2024-05-07 19:29:12,738] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: research_scraper.research_scraper manual__2024-05-07T19:29:10.005131+00:00 [queued]>
[2024-05-07 19:29:12,739] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-05-07 19:29:12,739] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-05-07 19:29:12,739] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-05-07 19:29:12,752] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): research_scraper> on 2024-05-07 19:29:10.005131+00:00
[2024-05-07 19:29:12,762] {standard_task_runner.py:52} INFO - Started process 4517 to run task
[2024-05-07 19:29:12,770] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'research_scraper', 'research_scraper', 'manual__2024-05-07T19:29:10.005131+00:00', '--job-id', '194', '--raw', '--subdir', 'DAGS_FOLDER/ResearchScraper.py', '--cfg-path', '/tmp/tmpszwyyjbf', '--error-file', '/tmp/tmpif0e0166']
[2024-05-07 19:29:12,771] {standard_task_runner.py:80} INFO - Job 194: Subtask research_scraper
[2024-05-07 19:29:13,244] {task_command.py:369} INFO - Running <TaskInstance: research_scraper.research_scraper manual__2024-05-07T19:29:10.005131+00:00 [running]> on host b29e3e3252f3
[2024-05-07 19:29:13,453] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=research_scraper
AIRFLOW_CTX_TASK_ID=research_scraper
AIRFLOW_CTX_EXECUTION_DATE=2024-05-07T19:29:10.005131+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-07T19:29:10.005131+00:00
[2024-05-07 19:29:13,468] {conn.py:381} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('192.168.247.6', 9092)]>: connecting to kafka:9092 [('192.168.247.6', 9092) IPv4]
[2024-05-07 19:29:13,469] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2024-05-07 19:29:13,471] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('192.168.247.6', 9092)]>: Connection complete.
[2024-05-07 19:29:13,579] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2024-05-07 19:29:13,580] {conn.py:1269} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2024-05-07 19:29:13,584] {logging_mixin.py:115} INFO - ['2']
[2024-05-07 19:29:13,584] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ResearchScraper.py", line 30, in call_scopus
    get_scopus_data()
  File "/opt/airflow/dags/scopus.py", line 64, in get_scopus_data
    start_pointer = read_start_pointer()
  File "/opt/airflow/dags/scopus.py", line 36, in read_start_pointer
    result[f'{row[0].strip()}{row[1].strip()}']= int(row[2].strip())
IndexError: list index out of range
[2024-05-07 19:29:13,609] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=research_scraper, task_id=research_scraper, execution_date=20240507T192910, start_date=20240507T192912, end_date=20240507T192913
[2024-05-07 19:29:13,622] {standard_task_runner.py:97} ERROR - Failed to execute job 194 for task research_scraper (list index out of range; 4517)
[2024-05-07 19:29:13,664] {local_task_job.py:156} INFO - Task exited with return code 1
[2024-05-07 19:29:13,713] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
